crates/oxc_ast/src/serialize/mod.rs

```diff
@@ -124,12 +124,38 @@ impl Program<'_> {
     const start = IS_TS ? 0 : DESER[u32](POS_OFFSET.span.start),
         end = DESER[u32](POS_OFFSET.span.end);
 
+
+    /* IF LINTER */
+    // Buffers will be cleaned up after the main program is deserialized.
+    // We hold references here in case the comments need to be later accessed.
+    let ref_uint32 = uint32;
+    let ref_uint8 = uint8;
+    let ref_sourceText = sourceText;
```

Comment:
Style nit: Our convention is snake_case in Rust code, but camelCase in JS. Maybe `uint32Ref` or `uint32Copy`?